dynatrace-oneagent-operator/values.yaml

```diff
@@ -63,6 +63,7 @@ oneagent:
   flavor: ""
 
 secret:
-  autoCreate: true
+  autoCreate: false
```

Comment:
We should create the secret by default if apiUrl is set. Don't if it isn't set.

Reply:
Made the changes - autoCreate set to true and altered If-statement in the secret file in order to make sure that it will be only created if autocreate is true and if there is a apiUrl set.